### PR TITLE
tui: Add software group selection

### PR DESCRIPTION
**Pull request by poncovka**
User is able to select which software groups to install in a text-based installation.

### CODE REVIEW

**Comment by jkonecny12** [comment]
jenkins, test this please

File: pyanaconda/ui/tui/spokes/software.py [hunk 1]
```diff
@@ -209,40 +292,45 @@ def ready(self):
...
-                self.payload.selectEnvironment(self.environment)
-                changed = True
-            elif self._origEnv != self.environment:
-                # environment changed, clear the list of packages and select the new
-                # one
-                self.payload.data.packages.groupList = []
+
+            # Changed the environment or addons, clear and setup
+            if not self._origEnv \
+                    or self._origEnv != self.environment\
```
Comment (snbueno) [inline pyanaconda/ui/tui/spokes/software.py]: Space between `self.environment` and `\` please.

**Comment by jkonecny12** [comment]
You can ignore the jenkins result here. Fixes for these are in progress.

File: pyanaconda/ui/tui/spokes/software.py (hunk 1 above)
Comment (poncovka) [inline pyanaconda/ui/tui/spokes/software.py]: Fixed.

**Comment by jkonecny12** [comment]
I'm giving POST-ACK here. Looks good to me.

**Pull request merged**